### ISSUE TITLE
Ender Generators(both) can't be broken. 1.12.x

### ISSUE THREAD

**Issue by antisocialian** [MoSadie/ExponentialPower#14]
I can't seem to break the Ender Generators, I've used many different ways. Several ways crash the game entirely. 

When I break them, the game looks like they are gone, but then when i put another block in its place, the Ender Generator shows up again as if it had never been broken. Its inventory is not dropped thru this process. (Seems like ghost blocks)

I'm playing a singleplayer, using version 1.3.1 of the mod on MC 1.12.1(forge 14.22.1.2478)

**Comment by MoSadie** [comment]
Hmm, I can't seem to recreate the issue with just this mod installed. What mods do you have installed, so I can better test your bug?

**Comment by antisocialian** [comment]
:/ sadly quite a few. Here is a dropbox link for the exported Twitch(Curse) pack, https://www.dropbox.com/s/u8tqc6krn2i2kos/my%20modpack-1.zip?dl=0

Otherwise here is the list of the files in the mods folder: https://pastebin.com/jZApna06

**Comment by antisocialian** [comment]
just realized i hadn't looked into the logs, it wasn't crashing so I didn't think to look. I am seeing this message: https://pastebin.com/bGJJ6rG6
Gonna do some testing with the Tesla files disabled and see what happens.

Edit: So I tried disabling everything that adds/needs the [Tesla Core Lib](https://minecraft.curseforge.com/projects/tesla-core-lib) and it still was doing it, very confused.
Looked at the page for the [Tesla Core Lib](https://minecraft.curseforge.com/projects/tesla-core-lib), see it requires [Tesla](https://minecraft.curseforge.com/projects/tesla). Tested by creating a new pack and adding [Industrial Foregoing](https://minecraft.curseforge.com/projects/industrial-foregoing) and it still does it. Added the [Tesla](https://minecraft.curseforge.com/projects/tesla) mod and tried again and it works fine. 

**tl;dr** Needed [Tesla](https://minecraft.curseforge.com/projects/tesla) installed too.

I don't know why refined storage was throwing an error about Tesla when I broke either of your generators. I'm going to say a "wizard did it" and call it fixed.

**Comment by MoSadie** [comment]
Ok then. I'm going to leave this open because I want to know why refined storage is upset, but I'm glad that adding Tesla fixed it!

**Comment by MoSadie** [comment]
Figured it out! Fixed in [v1.3.1.1](https://github.com/MoSadie/ExponentialPower/releases/tag/v1.3.1.1).
